routes/applications.js

```diff
@@ -2,30 +2,35 @@ const express = require("express");
 const pool = require("../db");
 const { verifyRole } = require("../middleware/authorization");
 const { ROLE_NAMES } = require("../utils")
+const { respondWithError, isUniqueConstraintViolation } = require("./helpers")
 const router = express.Router();
 router.use(express.json());
 
 router.get("/applications", verifyRole(ROLE_NAMES.mentor), async (req, res) => {
     try {
         const result = await pool.query("SELECT * FROM applications");
-        res.json(result.rows);
+        res.json({ applications: result.rows});
     } catch (err) {
         console.error(err);
-        res.status(500).send("Server Error");
+        respondWithError(res);
     }
 });
 
 router.post("/applications", async (req, res) => {
-    const { name, text } = req.body;
+    const { name, about, email } = req.body;
     try {
         const result = await pool.query(
-            "INSERT INTO applications (name, about) VALUES ($1, $2) RETURNING *",
-            [name, text]
+            "INSERT INTO applications (name, about, email) VALUES ($1, $2, $3) RETURNING *",
+            [name, about, email]
```

Comment:
can we change to insert the following fields:
id: number;
- name: string;
- email: string;
- about: string;
- languages: string[];
- photo: string; (optional)
- link to video: string; (optional)
- discord nickname: string;
- role: string;

Reply:
Yes, sure, but I think it should be done in a separate ticket: https://github.com/catbytes-community/webplatform-backend/issues/10
I can take it for the next week